Cambios en back de Cliente

RomanLeyva532 opened the pull request:
Cree el modelo de datos para usuario(nombreusuario,username,pass,email), así como parte del controlador de registro para guardar usuarios en la base de datos de mongo

RomanLeyva532 committed:
Cambios en back de Cliente

Cree el modelo de datos para usuario(nombreusuario,username,pass,email), así como parte del controlador de registro para guardar usuarios en la base de datos de mongo

ShabbyRug775 commented:
Los cambios parecen estar bien, ahora para continuar con el back de clientes por favor añade las rutas en la parte del servidor y si puedes realiza pruebas para ver si los datos se pasan de forma adecuada entre el servidor y el cliente.

ShabbyRug775 commented:
También añade los middlewares del servidor y los esquemas por favor.

RomanLeyva532 committed:
Rutas servidor, tokens y pruebas en back

Realice pruebas en el servidor donde el registro y login se desempeñan correctamente, sin embargo presente dificultades para mostrarlo en el front, lo que es el formulario de registro(de prueba) se ve en blanco.

ShabbyRug775 reviewed (commented):
Parece que todo está en orden, voy a realizar el merge a la rama principal.
